### DISCUSSION TITLE
query input box

### DISCUSSION THREAD

**Discussion by mohammadazeemwani** [scalar/scalar#2554] [Q&A]
when I have a query parameter in the api route, I want ti to show there a box in which i will pass the query and see the response right there

**Comment by hanspagel** [comment]
Thanks! Can you share a screenshot? I can’t really wrap my head around it yet.

**Comment by mohammadazeemwani** [comment]
![image](https://github.com/user-attachments/assets/ee49c233-1aa6-42cb-b89c-992e0a457826)
Hope this works!
